### ISSUE TITLE
Fetching stock ISO fails

### ISSUE THREAD

**Issue by mbr0wn** [gnuradio/gnuradio-livesdr#15]
```
/home/mbant/src/gnuradio-livesdr/iso/ubuntu-14.04.5-desktop-amd64.iso
[INFO] Downloading stock Ubuntu ISO image if needed...
[INFO] Checking for Ubuntu gpg public key (0xFBB75451)
[INFO] Retrieving SHA256 hashes and signature files
gpg: Signature made Thu 04 Aug 2016 01:46:28 PM PDT using DSA key ID FBB75451
gpg: Good signature from "Ubuntu CD Image Automatic Signing Key <cdimage@ubuntu.com>"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: C598 6B4F 1257 FFA8 6632  CBA7 4618 1433 FBB7 5451
gpg: Signature made Thu 04 Aug 2016 01:46:28 PM PDT using RSA key ID EFE21092
gpg: Can't check signature: public key not found
[ERROR] SHA256SUMS signature verification failed!
Makefile:28: recipe for target '/home/mbant/src/gnuradio-livesdr/iso/ubuntu-14.04.5-desktop-amd64.iso' failed
make: *** [/home/mbant/src/gnuradio-livesdr/iso/ubuntu-14.04.5-desktop-amd64.iso] Error 1
```

**Comment by mbr0wn** [comment]
Actually that was the output on the second run.

**Comment by jmcorgan** [comment]
Yeah, that's a bug in the script that doesn't deal well with a previous failure.  Deleting the .tmp file works, but I need to make it more robust to this condition.

**Comment by dl1com** [comment]
I ran into the same error on a fresh Ubuntu 16.04.2 live image.
Importing the Ubuntu archive keyring helped:

    gpg --import /usr/share/keyrings/ubuntu-archive-keyring.gpg

(see http://askubuntu.com/a/326486)

**Comment by jmcorgan** [comment]
As in with the current livesdr branch?  I can add that to the 'make deps' stage.

**Comment by dl1com** [comment]
@jmcorgan Yes, it happened with the latest version of the livesdr branch (d00ae583d2cac2b1d8c84301e9a8acd81f2b74a9), so adding it to the Makefile would make sense, I guess.

**Comment by jmcorgan** [comment]
This was fixed in 06d2abb.
